koinly-simple-export

```diff
@@ -9,6 +9,11 @@ set -euo pipefail
 [[ "$#" -lt 1 ]] && echo "koinly-simple-export <account>" && exit 1
 address=$1
 
+if [ "$1" == *".find"* ] || [ "$1" != "0x"* ]; then
```

Comment:
This is not needed the backend will remove .find if it is there and if the argument sent in is an address it will just return it.